libs/execution_context/ebpf_core.c

```diff
@@ -839,6 +839,63 @@ _ebpf_core_protocol_map_update_element(_In_ const ebpf_operation_map_update_elem
     EBPF_RETURN_RESULT(retval);
 }
 
+static ebpf_result_t
+_ebpf_core_protocol_map_update_element_batch(
+    _In_ const ebpf_operation_map_update_element_batch_request_t* request,
+    _Inout_ ebpf_operation_map_update_element_batch_reply_t* reply)
+{
+    EBPF_LOG_ENTRY();
+    ebpf_result_t retval;
+    ebpf_map_t* map = NULL;
+    size_t input_count = 0;
+    size_t output_count = 0;
+    size_t data_length;
+    size_t key_and_value_length;
+
+    retval = EBPF_OBJECT_REFERENCE_BY_HANDLE(request->handle, EBPF_OBJECT_MAP, (ebpf_core_object_t**)&map);
+    if (retval != EBPF_SUCCESS) {
+        goto Done;
```

Comment:
Was wondering if there was a reason why we don't have a trace message in such failures (especially in the execution_context code)?  For example, this location as well as the next couple of locations _could_ end up returning the same error code.  Specific trace messages in each error path would be of great help in post-mortem debugging.

We're already (unconditionally) logging entry/exit here, so such (conditional) traces wouldn't add much of an overhead, would they?

Reply:
Internally it will log an error from ebpf_reference_base_object_by_handle if the handle is invalid.